### DISCUSSION TITLE
TLS error: Unsupported protocol.

### DISCUSSION THREAD

**Discussion by popy2k14** [schwabe/ics-openvpn#1416] [Configuration problems] [answered]
To make issues more manageable, I would appreciate it if you fill out the following details as applicable:

# General information
1. Android Version: 11
2. Android Vendor/Custom ROM: Pixel Expierence 11
3. Device: Oneplus 6T (fatja)
4. Version of the app (version number/play store version/self-built):  play store version 0.7.25 - 0.7.29
 
# Description of the issue
Since version 0.7.25 (Openssl 3.0.0 Update) i cant connect to my Unifi USG which sadly running an old 2.3.2 openvpn version.
It has the latest firmware (10/2021) but Unifi dont include newer openvpn, so i am stuck with it.

The issue the app is reporting:
`
TLS error: Unsupported protocol. This typically indicates that client and server have no common TLS version enabled. This can be caused by mismatched tls-version-min and tls-version-max options on client and server. If your OpenVPN client is between v2.3.6 and v2.3.2 try adding tls-version-min 1.0 to the client configuration to use TLS 1.0+ instead of TLS 1.0 only
`

Adding the suggested "tls-version-min 1.0" doesn't help.
Downgrading to 0.7.24 or lower solves the issue for me.

Any hint's how i can get the newer versions of the app connected again to my Unifi device?

**Comment by schwabe** [comment, marked as answer]
You probably need enable compatibility with 2.3.0 in basic settings and might even need to enable security level insecure in TLS settings.

**Comment by popy2k14** [comment]
thx, activating those two options did the trick with 0.7.25 and newer.
